sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-student-loan-balances-queue.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO
+  sims.queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'student-loan-balances',
+    '{
+        "retry": 3,
+        "retryInterval": 180000,
+        "dashboardReadonly": false,
+        "cron": "0 17 * * *",
+        "cleanUpPeriod": 2592000000
```

Comment:
Is it set for 30 days? If the data will come once a month it may even remove the latest imported one, right?
Should we keep this one for 60 days maybe?

Reply:
Is there any input or consideration for this comment?